tests/jballerina-unit-test/src/test/resources/test-src/functions/functions_with_default_parameters.bal

```diff
@@ -270,6 +270,29 @@ public function sleep(int millis) = @java:Method {
     'class: "org.ballerinalang.test.utils.interop.Utils"
 } external;
 
+function testFuncWithComputedNameFieldInRecordLiteralForDefaultValue() {
+    int a = 10;
+    function (record {| int a;|} x = {a}) returns int y = value1;
```

Comment:
This is not a computed field?

Reply:
Please keep this (with the correct name) and add a separate test for variable name fields.